projects/plugins/beta/src/admin/notice.template.php

```diff
@@ -39,6 +39,19 @@
 				margin-bottom:1em;
 			}
 		</style>
+
+		<?php
+		if ( isset( $plugin ) ) {
+			$plugin = $plugin; // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
It would be better to do like [this](https://github.com/Automattic/jetpack/blob/693568d7721b8d7fe201f7059c76a16141b1cb2b/projects/plugins/beta/src/admin/show-needed-updates.template.php#L16-L19) so it's clear at the top of the file which "globals" are expected.